public/pinion/v1.2-standard/spec.json

```
@@ -1444,7 +1444,7 @@
         },
         {
             "ref": "JP2",
-            "description": "Recovery via USB-C connector",
+            "description": "USB boot mode jumper. Allows recovery via USB-C.\n\nSetting this jumper will force the CM4 into USB boot mode. See J3 for more\ndetails.\n\nThis jumper allows to force nRPIBOOT to GND.\n",
```

See J3, can we mention USB c plug J3?

---

The description file is genereated, but when I back-fill to my source I think we can adjust here directly.

```suggestion
            "description": "USB boot mode jumper. Allows recovery via USB-C (J3).\n\nSetting this jumper will force the CM4 into USB boot mode. See J3 for more\ndetails.\n\nThis jumper allows to force nRPIBOOT to GND.\n",
```